README.rst

```diff
@@ -6,10 +6,29 @@ ftw.linkchecker
 Introduction
 ============
 
+``ftw.linkchecker`` is an add-on for Plone installations. It is meant to be run
+as a cronjob regularly to find broken links and references within Plone sites.
+
+How does it work
+****************
+
+- The config file is inspected and the information extracted.
+- Each Plone site is dealt with separately with its info extracted.
+- All fields possibly containing links are analysed and link/relation like
+  strings are collected and stored as link objects with some additional info.
+- A check is done for each link/relation whether they are broken.
+- A report is generated in an Excel sheet.
+- The report is sent to the email addresses configured.
+- If a valid upload location is provided, the file is stored there additionally.
+
+Important note
+**************
+
 It's important, that this package isn't started by conjob in non productive
```

Comment:
cronjob :-)
unless you meant con as in 'conman' or 'convict' ;-)
Drop the commas
non-_production_ deployments
This is because the command ....

Reply:
And - I don't get the problem with a zope ctl command ???

Reply:
I create an issue regarding the question so we can go on with this PR.